### PR TITLE
Firmware compression

### PR DESCRIPTION
**Pull request by floion**

### CODE REVIEW

File: layers/meta-balena-raspberrypi/conf/layer.conf [hunk 1]
```diff
@@ -322,3 +322,8 @@ RPI_KERNEL_DEVICETREE_append_raspberrypi4-64 = " broadcom/bcm2711-rpi-400.dtb"
 RPI_KERNEL_DEVICETREE_append_raspberrypi4-64 = " broadcom/bcm2711-rpi-cm4.dtb"
 
 PREFERRED_VERSION_linux-raspberrypi = "5.10.%"
+
+FIRMWARE_COMPRESSION_raspberrypi4-64 ?= "1"
+
+# avoid pseudo abort due to iwlwifi firmware being cleaned outside of fakeroot - this should be a temporary workaround until https://github.com/balena-os/meta-balena/pull/2270 is merged
+IWLWIFI_FW_TOCLEAN = ""
```
Comment (klutchell) [inline layers/meta-balena-raspberrypi/conf/layer.conf:329]:
```suggestion
IWLWIFI_FW_TOCLEAN_raspberrypi4-64 = ""
```

**Comment by floion** [comment]
@balena-ci I self-certify!

**Comment by keenanjohnson** [comment]
Hey all! Thanks for pushing this through! When or what is the process for this to be officially released and available via the web UI? Thanks again!

**Comment by klutchell** [comment]
Hey @keenanjohnson, thanks for your patience with this!

I've opened a new issue to request a new release to production for Raspberry Pi CM4 IO Board. I can't provide an ETA at this time but you can follow that issue for progress.

**Comment by keenanjohnson** [comment]
Awesome! Thanks @klutchell !

**Pull request merged**